prac_05/wimbledon.py

```diff
@@ -0,0 +1,41 @@
+FILENAME = "wimbledon.txt"
+
+
+def main():
+    data = get_data(FILENAME)
+    champion_to_titles, countries = analyze_data(data)
+    display_data(champion_to_titles, countries)
+
+
+def get_data(FILENAME):
```

Comment:
Naming convention- "get" implies taking input from user. "Extract" would be more coherent